### PR TITLE
try to fix  Duplicate of files in directory (gogs#2254 )

### PR DESCRIPTION
**Pull request by novaeye**
https://github.com/gogits/gogs/issues/2254

### CODE REVIEW

File: tree_entry.go [hunk 1]
```diff
@@ -110,6 +110,7 @@ func (tes Entries) Sort() {
 
 type commitInfo struct {
 	id    string
+	entry string
```
Comment (Unknwon) [inline tree_entry.go]: Maybe use `entryName` should be better?

**Comment by Unknwon** [comment]
Thanks your PR!

File: tree_entry.go (hunk 1 above)
Comment (novaeye) [inline tree_entry.go]: /nod, u r right!

File: tree_entry.go [hunk 2]
```diff
@@ -109,9 +109,10 @@ func (tes Entries) Sort() {
 }
 
 type commitInfo struct {
-	id    string
-	infos []interface{}
-	err   error
+	id        string
```
Comment (Unknwon) [inline tree_entry.go]: Since `id` was only used for identify the object, and now we use `entryName`, I think it can be removed.

**Comment by Unknwon** [comment]
Thanks!

You should ping me after update, GitHub does not send me notification for that...

**Pull request merged**